rai/node/node.cpp

```diff
@@ -1051,7 +1102,7 @@ void rai::block_processor::process_receive_many (std::unique_lock<std::mutex> &
 					node.ledger.rollback (transaction, successor->hash ());
 				}
 			}
-			auto process_result (process_receive_one (transaction, block.first, block.second));
+			auto process_result (process_receive_one (transaction, block.first, block.second, !force));
```

Comment:
Using `!force` here is not clear that it's directly related to state blocks... can we do something more clear ?

Reply:
Forced state blocks are not validated in verify_state_blocks () function
Because of that we should set set validated_state_block as "false" for forced blocks (!force)

Reply:
But `!force` will evaluate to true when `force` is false which can happen for things other than state blocks (or verified blocks ?)

Reply:
Yes, but only state blocks vefirication can be skipped by flag in ledger.processor